packages/client/components/TaskEditor/CommentEditor.tsx

```diff
@@ -190,6 +190,12 @@ const CommentEditor = (props: Props) => {
     onBlur(e)
   }
 
+  useEffect(() => {
+    if (editorRef.current) {
+      editorRef.current.focus();
+    }
+  }, [editorRef, editorState]);
```

Comment:
-2 @gitstart I don't think you can put `editorState` directly into the `useEffect` dependency, it's a huge object and changes all the time, so it would cause lots of re-render.

![Google Chrome-Check-in #1  JimmyLv_吕立青’s Team-000077-20220831](https://user-images.githubusercontent.com/4997466/187674995-46b28772-bd98-4420-a72e-109ec3f359aa.gif)